src/LiveSplit.Delta/UI/Components/DeltaComponent.cs

```diff
@@ -156,6 +148,19 @@ public void Update(IInvalidator invalidator, LiveSplitState state, float width,
                 InternalComponent.TimeValue = null;
             }
 
+            var text = comparisonName;
+            if (Settings.OverrideText)
+            {
+                text = Settings.DifferentialText && InternalComponent.TimeValue < TimeSpan.Zero ? Settings.TextAhead : Settings.TextBehind;
```

Comment:
Does this mean you're using `TextBehind` when the runner is ahead if `DifferentialText` is off? That's pretty confusing. Maybe change the naming and/or add some comments to explain this behavior?

Reply:
That's the intended behavior (only the settings box for `TextBehind` is enabled when `DifferentialText` is off). I'll change the name to better signify that this variable acts as the overridden text when it is constant, or the text when the runner is behind when there's a different one for when the runner is ahead.